src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -0,0 +1,123 @@
+import static frc.robot.Constants.ClimbConstants.CLIMB_LIMIT_FORWARD;
+import static frc.robot.Constants.ClimbConstants.CLIMB_LIMIT_REVERSE;
+import static frc.robot.Constants.ClimbConstants.CLIMB_MAGNETIC_OFFSET;
+import static frc.robot.Constants.ClimbConstants.CLIMB_ROTOR_TO_SENSOR_RATIO;
+import static frc.robot.Constants.ClimbConstants.CLIMB_SLOT_CONFIGS;
+import static frc.robot.Constants.ClimbConstants.CLIMB_STATOR_CURRENT_LIMIT;
+import static frc.robot.Constants.ClimbConstants.CLIMB_SUPPLY_CURRENT_LIMIT;
+import static frc.robot.Constants.ClimbConstants.DEVICE_ID_CLIMB_DETECTION;
+import static frc.robot.Constants.ClimbConstants.DEVICE_ID_CLIMB_ENCODER_1;
+import static frc.robot.Constants.ClimbConstants.DEVICE_ID_CLIMB_MOTOR_1;
+import static frc.robot.Constants.ClimbConstants.DEVICE_ID_CLIMB_MOTOR_2;
+
+import com.ctre.phoenix6.CANBus;
+import com.ctre.phoenix6.SignalLogger;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.CANrange;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.MutAngle;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+
+public class ClimbSubsystem extends SubsystemBase {
+  private static final CANBus CANIVORE_BUS_NAME = null;
+
+  private final TalonFX leftMotor = new TalonFX(DEVICE_ID_CLIMB_MOTOR_1, CANIVORE_BUS_NAME);
+  private final TalonFX rightMotor = new TalonFX(DEVICE_ID_CLIMB_MOTOR_2, CANIVORE_BUS_NAME);
+
+  private final CANcoder leftClimbEncoder = new CANcoder(DEVICE_ID_CLIMB_ENCODER_1, CANIVORE_BUS_NAME);
+  private final CANcoder rightClimbEncoder = new CANcoder(DEVICE_ID_CLIMB_ENCODER_1, CANIVORE_BUS_NAME);
+
+  private final CANrange cageDetection = new CANrange(DEVICE_ID_CLIMB_DETECTION, CANIVORE_BUS_NAME);
+
+  private final VoltageOut climbControl = new VoltageOut(0.0).withEnableFOC(true);
+
+  private final StatusSignal<Angle> motorPosition = leftMotor.getPosition();
+
+  // private final MutableMeasure climbAngle = MutableMeasure.zero(Rotations);
+  private final MutAngle climbAngle = Radians.mutable(0);
+
+  private final SysIdRoutine m_sysIdRoutineTranslation = new SysIdRoutine(
+      new SysIdRoutine.Config(
+          null,
+          null,
+          null,
+          // Log state with SignalLogger class
+          state -> SignalLogger.writeString("SysIdTranslation_State", state.toString())),
+      new SysIdRoutine.Mechanism(
+          (volts) -> leftMotor.setControl(climbControl.withOutput(volts.in(Volts))),
+          null,
+          this));
```

Comment:
Let's just remove the SysId routine for now. Since we won't be running closed loop, we don't need SysId, so there's no reason to get this fixed up.